dist/profile/templates/azure.ci.jenkins.io/agents.yaml.erb

```diff
@@ -0,0 +1,497 @@
+        storageAccountType: "Standard_LRS"
+        templateDesc: "Dynamically provisioned Ubuntu 20.04 LTS machine"
+        templateDisabled: true
+        templateName: "ubuntu-jenkinsinfra-highmem"
+        usageMode: "Use this node as much as possible"
+        usePrivateIP: false
+        virtualMachineSize: "Standard_D16_v4"
+  - azureVM:
+      azureCredentialsId: "azure-credentials"
+      cloudName: "azure-eastus"
+      configurationStatus: "pass"
+      deploymentTimeout: 3000
+      maxVirtualMachinesLimit: 1
+      newResourceGroupName: "eastus-cijenkinsio"
+      resourceGroupReferenceType: "new"
+      vmTemplates:
+      - agentLaunchMethod: "SSH"
+        agentWorkspace: "C:\\Jenkins"
+        builtInImage: "Windows Server 2016"
+        credentialsId: "jenkinsvmagents-userpass"
+        diskType: "managed"
+        doNotUseMachineIfInitFails: false
+        enableMSI: false
+        enableUAMI: false
+        ephemeralOSDisk: false
+        executeInitScriptAsRoot: false
+        existingStorageAccountName: "<%= @agents_azure_vms_storage_account_windows %>"
+        imageReference:
+          galleryImageDefinition: "jenkins-agent-windows-2019"
+          galleryImageVersion: "<%= @agents_azure_vms_gallery_image_version %>"
+          galleryName: "prod_packer_images"
+          galleryResourceGroup: "prod-packer-images"
+          gallerySubscriptionId: "<%= @agents_azure_vms_gallery_subscription_id %>"
+        imageTopLevelType: "advanced"
+        installDocker: false
+        installGit: false
+        installMaven: false
+        javaPath: "java"
+        labels: "windows amd64 windock azure vm"
+        location: "East US"
+        maximumDeploymentSize: 10
+        noOfParallelJobs: 1
+        osDiskSize: 200
+        osType: "Windows"
+        preInstallSsh: true
+        retentionStrategy:
+          azureVMCloudRetentionStrategy:
+            idleTerminationMinutes: 30
+        shutdownOnIdle: false
+        storageAccountNameReferenceType: "new"
+        storageAccountType: "Standard_LRS"
+        templateDesc: "Dynamically provisioned Windows 2019 Core image"
+        templateDisabled: false
+        templateName: "win2019-jenkinsinfra"
+        usageMode: "Use this node as much as possible"
+        usePrivateIP: false
+        virtualMachineSize: "Standard_DS4_v2"
+      - agentLaunchMethod: "SSH"
+        agentWorkspace: "C:\\Jenkins"
+        builtInImage: "Windows Server 2016"
+        credentialsId: "jenkinsvmagents-userpass"
+        diskType: "managed"
+        doNotUseMachineIfInitFails: false
+        enableMSI: false
+        enableUAMI: false
+        ephemeralOSDisk: false
+        executeInitScriptAsRoot: false
+        existingStorageAccountName: "<%= @agents_azure_vms_storage_account_windows %>"
+        imageReference:
+          id: "/subscriptions/<%= @agents_azure_vms_gallery_subscription_id %>/resourceGroups/prod-packer-images-eastus/providers/Microsoft.Compute/images/jenkins-agent-win10"
```

Comment:
where does this image come from? I can't see it in packer, the reason I ask is that I was wondering why this still used a managed image instead of image gallery

Reply:
I have no idea where the Windows 10 image comes from, neither where it's used :| I would vote to remove it, but it might break something. Asked in IRC

Reply:
cc @MarkEWaite @oleg-nenashev @olblak @jglick : do you know which job are expected to use the Windows 10 agents or if we can "just" remove this in favor of Windows 2019 ?

Reply:
I had not heard of Windows 10 agents. For CI I would certainly expect to be using the latest available version of Windows Server, either LTSC (2019) or SAC (whatever).

Reply:
Thanks for your answers and your time @jglick !